bin/test.sh

```diff
@@ -87,8 +90,23 @@ run_tests() {
     popd
 }
 
-build_universe;
+# Check env early, before starting the cluster:
+if [ -z "$TEST_RUNNER_DIR" -o -z "$DOCKER_IMAGE" \
+        -o -z "$DCOS_USERNAME" -o -z "$DCOS_PASSWORD" \
+        -o -z "$STUB_UNIVERSE_URL" \
+        -o -z "$AWS_ACCESS_KEY_ID" -o -z "$AWS_SECRET_ACCESS_KEY" \
+        -o -z "$S3_BUCKET" -o -z "$S3_PREFIX" ]; then
+    echo "Missing required env. See check in ${BIN_DIR}/test.sh."
+    env
+    exit 1
+fi
+
+_notify_github pending "Starting Cluster"
```

Comment:
What's the purpose of these notifications?  Anyone curious about the progress of the build can instead look at the progress in Jenkins, right?